public/pages/CreateMonitor/components/VisualGraph/utils/helpers.js

```diff
@@ -41,7 +41,7 @@ import {
 import { MONITOR_TYPE } from '../../../../../utils/constants';
 
 export function getYTitle(values) {
-  return _.get(values, 'fieldName[0].label', 'count');
+  return _.get(values, `aggregations[0].fieldName`, 'doc_count');
```

Comment:
What happens when the monitor is not an aggregation monitor?

Reply:
This function is actully only used for query-level monitor, which supports 0 or 1 metric aggregation. `aggregations[0].fieldName` would be the only metric defined, and if undefined we use `doc_count` (count of documents) as the aggregation source.